samples/Microsoft.Azure.WebJobs.Extensions.OpenApi.FunctionApp.Filters/VersionPrefixDocumentFilter.cs

```diff
@@ -0,0 +1,34 @@
+using System.Linq;
+using System.Text.RegularExpressions;
+
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Abstractions;
+using Microsoft.Azure.WebJobs.Extensions.OpenApi.Core.Extensions;
+using Microsoft.OpenApi.Models;
+
+namespace Microsoft.Azure.WebJobs.Extensions.OpenApi.FunctionApp.Filters
+{
+    /// <summary>
+    /// This is an example implementation for a document filter to demonstrate the ability of modifying the IDocumentFilter
+    /// after it has been created.
+    ///
+    /// This filter prepends all paths in the OpenApi document when a request with query string like ?tag=v1,name is made.
+    /// </summary>
+    public class VersionPrefixDocumentFilter : IDocumentFilter
+    {
+        public void Apply(OpenApiDocument document, IHttpRequestDataObject request)
+        {
+            var tags = request.Query["tag"].ToArray(",");
```

Comment:
The following querystrings are all valid:

* `tag=v1,name`
* `tag=v1&tag=name`
* `tag=[v1,name]`
* `tag[]=v1,name`

Therefore, using `ToArray(",")` may not be appropriate here.

Reply:
I use this file to test the document filter and demonstrate the idea, do you want to keep this file? If so, I can improve this code, but if I'm honest I think this is more a distraction. It is probably better to simply remove the file entirely and document that `IDocumentFilter` is available for implementing document post-processing.

Reply:
Oh, you're right. I agree.